src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -24,16 +24,19 @@ public ConsoleLogger(bool verbose, string toolName, IEnumerable<FailureLevel> le
 
         public string CapturedOutput => _capturedOutput?.ToString();
 
-        public bool Verbose { get; set; }
+        private readonly bool _quiet;
 
-        private void WriteLineToConsole(string text = null)
+        private void WriteLineToConsole(string text = null, bool ignoreQuiet = false)
```

Comment:
>ignoreQuiet [](start = 65, length = 11)

let's use something 'forceEmitOfErrorMessages'

Reply:
Done.

---
In reply to: [559737163](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r559737163) [](ancestors = 559737163)